tests/clipping/main.nf.test

```diff
@@ -0,0 +1,25 @@
+nextflow_pipeline {
+
+    name "Clipping"
+    script "main.nf"
+
+    test("Should run with clipping params") {
+        when {
+            params {
+                aligner = "bismark"
```

Comment:
This is the default, so probably not needed..

Reply:
Looking at this fresh I'm going back on my previous opinion, as I think it makes it clearer to read even if it's not needed 😅  (now I have several of these files open at once)